### PR TITLE
fix: external links drawer-UI fix

### PR DESCRIPTION
**Pull request by rish2320**
# Description
External Links Drawer was broken fixed it.

Fixes #[4714](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/4714)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Tested the cases manually.

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
         height: 100%;
         border-radius: 0;
         margin-top: 0;
-        padding: 16px 0;
+        padding: 0;
```
Comment (milindk8) [inline src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss:9]: if there is no default padding then remove this padding instead assigning 0

File: src/components/externalLinks/externalLinks.component.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@
         height: calc(100% - 40px);
         border-radius: 0 0 8px 8px;
         margin-top: 0;
-        padding: 16px 0;
+        padding: 0;
```
Comment (milindk8) [inline src/components/externalLinks/externalLinks.component.scss:8]: same here

**Review by milindk8 — APPROVED** [review]

File: src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss (hunk 1 above)
Comment (rish2320) [reply src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss:9]: a default paading was defined so to counter it.

File: src/components/externalLinks/externalLinks.component.scss (hunk 2 above)
Comment (rish2320) [reply src/components/externalLinks/externalLinks.component.scss:8]: same here.

File: src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss [hunk 3]
```diff
@@ -104,6 +105,7 @@
                             flex-direction: column;
 
                             input {
+                                width: 604px;
```
Comment (vivek-devtron) [inline src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss:108]: can't this be derived from the parent div?

File: src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss [hunk 4]
```diff
@@ -138,6 +140,7 @@
                     flex-direction: column;
 
                     textarea {
+                        width: 604px;
```
Comment (vivek-devtron) [inline src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss:143]: Same here

File: src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss (hunk 1 above)
Comment (rish2320) [reply src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss:9]: done

File: src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss (hunk 3 above)
Comment (rish2320) [reply src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss]: done

File: src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss (hunk 4 above)
Comment (rish2320) [reply src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss]: done

**Review by vivek-devtron — APPROVED** [review]

File: src/components/app/details/triggerView/cdMaterial.tsx [hunk 5]
```diff
@@ -1046,10 +1046,10 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
 
     renderTriggerModalCTA(isApprovalConfigured: boolean) {
         const buttonLabel = CDButtonLabelMap[this.props.stageType]
-        const hideConfigDiffSelector = isApprovalConfigured && this.props.material.length <= 1
         const disableDeployButton =
             this.isDeployButtonDisabled() ||
             (this.props.material.length > 0 && this.isImageApprover(this.props.material[0]?.userApprovalMetadata))
+        let hideConfigDiffSelector = isApprovalConfigured && disableDeployButton
```
Comment (vivek-devtron) [inline src/components/app/details/triggerView/cdMaterial.tsx:1052]: Use const

**Review by vivek-devtron — APPROVED** [review]

File: src/components/app/details/triggerView/cdMaterial.tsx (hunk 5 above)
Comment (rish2320) [reply src/components/app/details/triggerView/cdMaterial.tsx]: done

**Pull request merged**